docs/CHANGELOG.rst

```diff
@@ -24,6 +24,11 @@ jax 0.2.8 (Unreleased)
   * ``host_callback.outfeed_receiver`` has been removed (it is not necessary,
     and was deprecated a few months ago).
 
+* Breaking changes:
+
+  * ``jax.numpy.pad`` now take keyword arguments. Extra parameter (``constant_values``)
```

Comment:
I would propose "positional argument ``constant_values`` has been removed. ..."

Reply:
Thanks for the proposal.
Updated! PTAL.